docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -8,6 +8,57 @@ sidebar_label: Time Series
 
 Below functions are used for time series analysis.
 
+## seriesOutliersTukey
+
+Detects a possible anomaly in series using [Tukey Fences](https://en.wikipedia.org/wiki/Outlier#Tukey%27s_fences).
+
+**Syntax**
+
+``` sql
+seriesOutliersTukey(series);
+```
+
+**Arguments**
+
+- `series` - An array of numeric values
+
+**Returned value**
+
+- Returns an array of the same length where each value represents a modified Z-score of possible anomaly of corresponding element in the series.
```

Comment:
Let's move the description in l. 27+27 to l. 14 and rather say here sth. shorter like `An array of outlier scores`.

Also, the term `Z-score` needs either explanation, a link to an explanation (Wikipedia) or replacement by something more common (e.g. just `outlier score`).